python/tvm/relax/frontend/torch/fx_translator.py

```diff
@@ -526,6 +526,22 @@ def _einsum(self, node: fx.node.Node) -> relax.Var:
             return self.block_builder.emit(relax.op.einsum(tuple(args[1]), args[0]))
         return self.block_builder.emit(relax.op.einsum(args[1:], args[0]))
 
+    def _unbind(self, node: fx.node.Node) -> relax.Var:
```

Comment:
Pplease also include the unit test in the frontend test.
https://github.com/apache/tvm/blob/main/tests/python/relax/test_frontend_from_fx.py

Reply:
done